.idea/allotropy.iml

```diff
@@ -6,7 +6,7 @@
       <sourceFolder url="file://$MODULE_DIR$/tests" isTestSource="true" />
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.11 (benchling-allotropy)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.11 (Binding_affinity)" jdkType="Python SDK" />
```

Comment:
Thanks for pointing this out, Nate. This issue occurred because we configured our own interpreter, and Python automatically made the change. We've made the necessary adjustments and pushed the updated code.